packages/frontend/src/components/common/PivotTable/ValueCell.tsx

```diff
@@ -110,6 +112,7 @@ const ValueCell: FC<ValueCellProps> = ({
             onCopy={handleCopy}
             onOpen={() => setIsMenuOpen(true)}
             onClose={() => setIsMenuOpen(false)}
+            appliedDashboardFilters={appliedDashboardFilters}
         >
             <Cell
```

Comment:
It's definitely possible that if we need more perf in this table in the future, we'll want to stop creating one of these menus for each cell. That's a lot of dom to add for something that is duplicated everywhere.

Reply:
100%! such a great idea @magnew